VideoRenderer/VideoRenderer/VideoStreamViewController.swift

```diff
@@ -34,18 +34,22 @@ class VideoStreamView: UIView {
         return track.naturalSize
     }
     
-    var resizeOptions = ResizeOptions(allowVerticalBars: true, allowHorizontalBars: true) {
+    var resizeOptions = ResizeOptions(allowVerticalBars: true, allowHorizontalBars: true, resizeAspectFill: false) {
         didSet {
-            guard let size = naturalSize else { return }
-            
-            playerLayer?.videoGravity =
-                resizeOptions.videoGravity(for: size, in: bounds.size)
+            if resizeOptions.resizeAspectFill {
+                playerLayer?.videoGravity = AVLayerVideoGravityResizeAspectFill
+            } else {
+                let size = naturalSize ?? CGSize.zero
+                playerLayer?.videoGravity =
+                    resizeOptions.videoGravity(for: size, in: bounds.size)
+            }
         }
     }
     
     struct ResizeOptions {
         let allowVerticalBars: Bool
         let allowHorizontalBars: Bool
+        let resizeAspectFill: Bool
```

Comment:
Why you need `resizeAspectFill` - it can be described with `allowVerticalBars  == false && allowHorizontalBars == false` isn't it?